pennylane/ops/qubit/matrix_ops.py

```diff
@@ -74,11 +74,12 @@ class QubitUnitary(Operation):
     def __init__(
         self, U, wires, do_queue=True, id=None, unitary_check=False
     ):  # pylint: disable=too-many-arguments
-        wires = Wires(wires)
-
         # For pure QubitUnitary operations (not controlled), check that the number
         # of wires fits the dimensions of the matrix
-        if not isinstance(self, ControlledQubitUnitary):
+
+        if not isinstance(self, qml.ControlledQubitUnitary):
```

Comment:
Since we've gotten rid of the inheritance between `QubitUnitary` and `ControlledQubitUnitary`, we can get rid of this check and un-indent all the following code.